crane-cloud/frontend
# add admin project management (disable/enable)

# Description

Enable disabling and enabling of projects. 
Integrated project details on the admin side.
Added live project logs on the admin side.

## Type of change



## Trello Ticket ID
https://trello.com/c/YXjm3zxQ
https://trello.com/c/0G73cIUi
https://trello.com/c/gmHr5xsR


## How Can This Been Tested?
Log in as admin, Navigate to projects and it  should display projects details and logs.
Disable one of the projects on the admin side and check it out on the client side to view possible actions on it

## Checklist:


# Screenshots

![image](https://user-images.githubusercontent.com/69305400/236896523-75ecc6a1-fcce-4598-99d1-f0ba7ff4245d.png)

![image](https://user-images.githubusercontent.com/69305400/236896593-4e80eeea-fb5a-44da-8d68-05bf417e9482.png)

Admin disabled project
![image](https://user-images.githubusercontent.com/69305400/236896647-3d00c9bc-a06b-4e07-a695-f76941565d08.png)
